SpatialGDK/Source/SpatialGDK/Private/Tests/TestRoutingWorker.cpp

```diff
@@ -204,7 +204,8 @@ class ConnectionHandlerStub : public AbstractConnectionHandler
 		check(!HasNextOpList());
 		TArray<TArray<OpList>> NewListsOfOpLists;
 		TArray<OpList> OpLists;
-		OpLists.Add(Builder.Move().CreateOpList());
+		OpLists.Add(MoveTemp(Builder).CreateOpList());
```

Comment:
Is MoveTemp() necessary here?

Reply:
Yes, have commented elsewhere. It's a pattern used for functions that release resources.